contracts/modules/viewer/NexusViewer.sol

```diff
@@ -0,0 +1,106 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.18;
+
+import {Multicall} from "../../abstract/Multicall.sol";
+import {IAssessment} from "../../interfaces/IAssessment.sol";
+import {IAssessmentViewer} from "../../interfaces/IAssessmentViewer.sol";
+import {IGovernance} from "../../interfaces/IGovernance.sol";
+import {INXMMaster} from "../../interfaces/INXMMaster.sol";
+import {IPooledStaking} from "../../interfaces/IPooledStaking.sol";
+import {IStakingViewer} from "../../interfaces/IStakingViewer.sol";
+import {ITokenController} from "../../interfaces/ITokenController.sol";
+
+/// @title NexusViewer Contract
+/// @notice This contract provides a unified view of system-wide data from various contracts within the Nexus Mutual platform.
+contract NexusViewer is Multicall {
+  struct LegacyPoolStake {
+    uint deposit;
+    uint reward;
+  }
+
+  struct ClaimableNxm {
+    uint governanceRewards;
+    IStakingViewer.AggregatedTokens aggregateStakingTokens;
+    IAssessmentViewer.AssessmentRewards assessmentRewards;
+    LegacyPoolStake legacyPooledStake;
+    uint v1CoverNotesAmount;
+  }
+
+  struct LockedNxm {
+    IStakingViewer.AggregatedTokens aggregatedTokens;
+    uint assessmentStakeAmount;
+  }
+
+  INXMMaster public immutable master;
+  IStakingViewer public immutable stakingViewer;
+  IAssessmentViewer public immutable assessmentViewer;
+
+  constructor(INXMMaster _master, IStakingViewer _stakingViewer, IAssessmentViewer _assessmentViewer) {
+    master = _master;
+    stakingViewer = _stakingViewer;
+    assessmentViewer = _assessmentViewer;
+  }
+
+  /// @notice Retrieves the claimable NXM tokens for a given member.
+  /// @dev This does not include NXM from manager rewards (use stakingViewer.getManagerTokenRewards)
```

Comment:
we should include this, why do a separate call for it and have the UI manage this customly?

Reply:
from what I understood with my discussion with @mixplore we want to separate it so that we don't call the manager specific calls for non pool managers.

@mixplore can you confirm how the FE prefers to call getClaimableNxm (with or without manager rewards)?

Reply:
it's inaccurate to display total claimable NXM without including the manager rewards in the dashboard. the separation makes sense for individual staking related calls, but not for aggregate data.

Reply:
@rackstar It should include the manager rewards if they exist aka the member is a pool manager